nlisim/cli.py

```diff
@@ -39,6 +39,7 @@ def run(obj, target_time: float) -> None:
 
     with tqdm(
         desc='Running simulation',
+        unit='step',
```

Comment:
It will actually show simulation time in hours rather than time step.

Reply:
Maybe I'm misunderstanding the semantics of simulation iteration. The README states:
> Now run simulation up to 50 timesteps using the first example config.

How should we refer to each full iteration of the simulation?

Reply:
Yeah, the README isn't accurate.  The issue is there isn't necessarily a well defined time step globally.  See https://github.com/NutritionalLungImmunity/nlisim/pull/89.

Reply:
Ok, for now, if we say something like
> Run the simulation for 50 __

What is the best description of "__"? "iterations"? "steps"? "timesteps"? "hours"?

Reply:
My understanding is expressed in units of hours.

Reply:
Ok, thanks. Closing this in favor of #98.